test/cql-pytest/cassandra_tests/validation/operations/select_test.py

```diff
@@ -2407,16 +2407,16 @@ def testFilteringOnDurationColumn(cql, test_keyspace):
         #assert_invalid_message(cql, table, "IN predicates on non-primary-key columns (d) is not yet supported",
         #                     "SELECT * FROM %s WHERE d IN (1s, 2s) ALLOW FILTERING")
 
-        assert_invalid_message(cql, table, "Slice restrictions are not supported on duration columns",
+        assert_invalid_message(cql, table, "unordered",
```

Comment:
This will probably make the same test fail on Cassandra. Can you check a different word (e.g., "duration")?
Or use assert_invalid_message_re and use a regular expression to check for two options.

Reply:
Certainly.

Reply:
Done.